main/integration-tests/README.md

```diff
@@ -14,25 +15,37 @@ To run integration tests, the following resources need to be created in advance:
 
 #### Resources to create through SWB UI:
 
-* **Test Administrator:** Create an internal admin-role user for running integration tests. (**Note the username and password**)
+- **Test Administrator:** Create an internal admin-role user for running integration tests. (**Note the username and password**)
 
-* **Test Project:** Create a default project for running integration tests. (**Note the projectId**)
+- **Test Project:** Create a default project for running integration tests. (**Note the projectId**)
 
-* **AWS Budget:** Create a budget for the AWS account associated with the Test Project.
+- **AWS Budget:** Create a budget for the AWS account associated with the Test Project.
+
+#### Resources for Advanced Tests
+
+- **Environment Type:** Import each specific Service Catalog Product as an Environment Type
+
+- **Environment Configuration:** Create a configuration for every corresponding environment type that was created
+
+- **Data Source Study:** If testing BYOB studies, create an external BYOB study on the deployment with Read & Write Access
 
 #### Resources to create through AWS Console:
 
-* **Test Administrator password:** Create an AWS Parameter Store record to store Test Administrator password. Pick a parameter name of your choice. Use 'SecureString' for parameter type 'text' for data type.(**Note the parameter name**)
+- **Test Administrator password:** Create an AWS Parameter Store record to store Test Administrator password. Pick a parameter name of your choice. Use 'SecureString' for parameter type 'text' for data type.(**Note the parameter name**)
```

Comment:
**Test Administrator password:** Create an AWS Parameter Store record to store Test Administrator password. Speciify the parameter name. Use 'SecureString' for parameter type 'text' for data type.(**Note the parameter name**)

"Use 'SecureString' for parameter type 'text' for data type." - This line is not clear.